chia/rpc/wallet_rpc_api.py

```diff
@@ -122,6 +125,8 @@ def get_routes(self) -> Dict[str, Endpoint]:
             "/get_next_address": self.get_next_address,
             "/send_transaction": self.send_transaction,
             "/send_transaction_multi": self.send_transaction_multi,
+            "/spend_clawback_coins": self.spend_clawback_coins,
```

Comment:
you should also add these to `wallet_rpc_client.py`

Reply:
added